webcompat/static/js/lib/bugform.js

```diff
@@ -428,8 +428,10 @@ function BugForm() {
     copy over the URL of a newly uploaded image asset to the bug
     description textarea.
   */
-  this.addImageURL = function(url) {
-    var imageURL = ['![Screenshot Description](', url, ')'].join('');
+   this.addImageURL = function(url) {
+    var file_ext = url.split('.').pop();
+    var thumb_url = url.replace('.' + file_ext, '-thumb.' + file_ext);
```

Comment:
Aren't we getting back the `thumb_url` path back from the response sent back by the `@uploads` `/` route?

Why not just make `addImageURL` take a response argument, rather than the url, that way you can grab `response.url` and `response.thumb_url` without the string manipulation.